package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "@yext/chat-ui-react",
-  "version": "0.8.2",
+  "version": "0.8.3",
   "description": "A library of React Components for powering Yext Chat integrations.",
   "author": "clippy@yext.com",
   "main": "./lib/commonjs/src/index.js",
   "module": "./lib/esm/src/index.mjs",
-  "types": "./lib/esm/src/index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./lib/esm/src/index.mjs",
-      "require": "./lib/commonjs/src/index.js"
+      "import": {
+        "default": "./lib/esm/src/index.mjs",
+        "types": "./lib/esm/index.d.ts"
+      },
+      "require": {
```

Comment:
does this issue occur on other lower level chat packages?

Reply:
it does not, according to arethetypeswrong. I think possibly because the libraries are simpler and don't need unique exports? not sure though